scripts/cli.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Outputs the contetns for content/reference/api/cli.md from the locally installed go-ipfs
+
+echo "*Generated on $(date +"%Y-%m-%d %T"), from go-ipfs $(ipfs version -n).*"
```

Comment:
My only comment would be to reference this script in the output so that people can find how to generate it again right away.

Reply:
Done!